src/akkd/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import styles from './Accordion.module.css'
+import {AccordionContent} from './AccordionContent'
+import {AccordionPanel} from './AccordionPanel'
+import {AccordionTitle} from './AccordionTitle'
+
+interface Panel {
+	key: any,

```

Comment:
Try to avoid using `any`, it's usually avoidable. `key` can _probably_ be `React.Key`, and `content` is probably `React.ReactNode`.